android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
         versionName "1.12"
         vectorDrawables.useSupportLibrary = true
         missingDimensionStrategy 'react-native-camera', 'general'
+        //multiDexEnabled true
```

Comment:
Either eliminate multidex support and increase the `minSdkVersion` to 21 (and test) or leave `minSdkVersion` as is and enable multidex support